### PR TITLE
Bump version to 0.4.0

### PR DESCRIPTION
**Pull request by mickel8**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,17 +33,19 @@ defmodule Resampling.Pipeline do
...
         output_caps: %Raw{channels: 2, format: :f32le, sample_rate: 44_100}
       },
-      file_sink: %File.Sink{location: "/tmp/out.raw"},
+      file_sink: %File.Sink{location: "/tmp/output.raw"},
+    ]
+
+    links = [
+      link(:file_src) 
+      |> to(:converter),
+      |> to(:file_sink)
```
Comment (mickel8) [inline README.md:47]: Honestly I didn't test it, but probably I should?

Comment (mat-hek) [reply README.md]: I would be great ;) you can check if there's already a similar pipeline in tests

**Review by mat-hek — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mickel8) [reply README.md]: Syntax was almost correct but I am not sure what we expect as output. Checking resulting file with `ffprobe` gives me this
```
Input #0, amrnb, from 'beep-f32le-44kHz-stereo.raw':
  Duration: 00:27:01.98, bitrate: 11 kb/s
    Stream #0:0: Audio: amr_nb, 8000 Hz, mono, flt
```

I suspect it shouldn't be 8kHz and mono. It also isn't playable.

Comment (mat-hek) [reply README.md]: A raw audio file contains just audio data without information about its shape, so FFmpeg tries to deduce it, but fails in this case. Try `ffplay -f f32le -ar 44100 -ac 2 output.raw`

**Pull request merged**